CHANGELOG.md

```diff
@@ -6,11 +6,14 @@
   deletes backup sidecar's ReplicaSet and creates new Deployment for sidecar.
   If upgrading failed for any reason, cluster TPR's `status.phase` will be FAILED.
   Recreate of the cluster TPR is needed on failure case.
+- Self-hosted cluster needs to be recreated. This doesn't affect non self-hosted cluster.
```

Comment:
we can still keep the old anti affinity label to new pods. so we effectively add new things without breaking old behavior.